src/components/firstProfile/uploadPhotos/UploadPhotos.tsx

```diff
@@ -110,6 +114,15 @@ const useStyles = makeStyles()(() => ({
     flexWrap: 'wrap',
     gap: 20,
     margin: 20,
+    '@media (max-width: 600px)': {
+      maxWidth: '380px',
+      width: '95vw',
+      height: 'auto',
```

Comment:
It looks that it make sense to move `   width: '95vw', height: 'auto',` to `@media (max-width: 368px)`